packages/flutter/lib/src/widgets/focus_traversal.dart

```diff
@@ -930,9 +930,6 @@ class DirectionalFocusAction extends _RequestFocusActionBase {
   /// The [LocalKey] that uniquely identifies this action to [DirectionalFocusIntent].
   static const LocalKey key = ValueKey<Type>(DirectionalFocusAction);
 
-  /// The direction in which to look for the next focusable node when invoked.
-  TraversalDirection direction;
```

Comment:
This field is referenced in the doc comment of the class. Can you update that please?